neuromation/cli/job.py

```diff
@@ -270,6 +270,12 @@ def job() -> None:
     show_default=True,
     help="Upload neuro config to the job",
 )
+@option(
+    "--wait-for-job-quota/--no-wait-for-job-quota",
```

Comment:
The name is quite long.
I have no idea what should be better.
Maybe @mariyadavydova can suggest something?

Reply:
I can suggest the following alternatives:

`--wait-for-quota`
`--queue-job`
`--fail-on-quota/--no-fail-on-quota`
`--keep-pending`
`--no-quota-error`
`--wait-for-seat`
`--run-eventually`

But as for me, none of them are good.

Reply:
I like `--wait-for-seat`.

Reply:
Ok, let's rename to `--wait-for-seat` and merge